### ISSUE TITLE
Automate Spiked Pit Trap Attack

### ISSUE THREAD

**Issue by sIKE23** [sIKE23/Mage-Wars#241]

**Comment by sIKE23** [comment]
Changed this from Spiked Pit Trap Attack to all of the Domination Zone effects while testing with Intangible0 tonight and he asked to test the Spiked Pit Traps' attack, I of course told him it wasn't automated at this point. It got me to thinking this would be a cool feature and opened up the issue for just the SPT. After further thought I think we should (within reason/if possible) automate the Zone Effects where possible, This of course would be after the

**Comment by ACG8** [comment]
Yes, zone effects are one of the things that we should definitely automate. Shouldn't be too hard; just check to see if the card is in the zone, and then apply its effect to everything else in the zone.
